packages/iTwinUI-react/src/core/utils/components/VirtualScroll.tsx

```diff
@@ -396,8 +401,8 @@ export const useVirtualization = (props: VirtualScrollProps) => {
     } as React.HTMLAttributes<HTMLElement>,
     innerProps: {
       style: { willChange: 'transform' },
-      ref: parentRef,
-    } as React.HTMLAttributes<HTMLElement>,
+      ref: mergeRefs(parentRef),
```

Comment:
Maybe add a comment here because `mergeRefs` looks unnecessary at first glance

something like this:
```suggestion
      ref: mergeRefs(parentRef), // converts object ref into callback ref for better types
```

Reply:
Added